CalibMuon/DTCalibration/plugins/DTVDriftSegment.cc

```diff
@@ -49,18 +54,34 @@ namespace dtCalibration {
 
   void DTVDriftSegment::setES(const edm::EventSetup& setup) {
     // Get the map of vdrift from the setup
-    ESHandle<DTMtime> mTime;
-    setup.get<DTMtimeRcd>().get(mTime);
-    mTimeMap_ = &*mTime;
+    if (readLegacyVDriftDB) {
+      ESHandle<DTMtime> mTime;
+      setup.get<DTMtimeRcd>().get(mTime);
```

Comment:
please move from `edm::EventSetup::get` to `edm::EventSetup::getData`
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#Getting_data_from_EventSetup_wit

Reply:
OK, but this migration has to be done on these classes anyhow, it is not needed as a consequence of this PR. 
Therefore it would make more sense to make it in its own PR, after this one is merged.
I can make a new PR for the migration right away after this one is merged; this is my preference given that this PR is by now tested and fully signed.